protected/humhub/modules/admin/models/forms/AuthenticationSettingsForm.php

```diff
@@ -60,8 +47,8 @@ public function init()
         $this->allowGuestAccess = $settingsManager->get('auth.allowGuestAccess');
         $this->showCaptureInRegisterForm = $settingsManager->get('auth.showCaptureInRegisterForm');
         $this->defaultUserProfileVisibility = $settingsManager->get('auth.defaultUserProfileVisibility');
-        $this->registrationApprovalMailContent = $settingsManager->get('auth.registrationApprovalMailContent', Yii::t('AdminModule.user', self::defaultRegistrationApprovalMailContent));
-        $this->registrationDenialMailContent = $settingsManager->get('auth.registrationDenialMailContent', Yii::t('AdminModule.user', self::defaultRegistrationDenialMailContent));
+        $this->registrationApprovalMailContent = $settingsManager->get('auth.registrationApprovalMailContent', ApproveUserForm::getDefaultApprovalMessage());
+        $this->registrationDenialMailContent = $settingsManager->get('auth.registrationDenialMailContent', ApproveUserForm::getDefaultApprovalMessage());
```

Comment:
Why is there two `ApproveUserForm::getDefaultApprovalMessage()`? Shouldn't a `ApproveUserForm::getDefaultDenialMessage()` also be created?

Reply:
In the lease `getDefaultDeclineMessage` as used in below code snippets.

Reply:
Thanks for the review. Typo is fixed!